<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordDialog</class>
 <widget class="QDialog" name="RecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A new record</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewRecord">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wow! A new record in this type of board.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameRecord">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameRecord">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert your name in the ranking:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeRecord">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>59</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarningEmptySpaces">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(All empty spaces will be deleted)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RecordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RecordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
